api/src/main/kotlin/io/sharing/server/api/user/adapter/inp/web/UserInactiveView.kt

```diff
@@ -0,0 +1,5 @@
+package io.sharing.server.api.user.adapter.inp.web
+
+class UserInactiveView(
```

Comment:
유스케이스와 같은 조합형식을 따라주는게 더 일관성있을거같습니다

```suggestion
class InactivateUserView(
```

Reply:
넵 피드백 주신내용 토대로 다시 수정하겠습니다. 감사합니다.